### ISSUE TITLE
Stage 1 decorators warning followed by complete meltdown

### ISSUE THREAD

**Issue by allthesignals** [machty/ember-concurrency-decorators#29]
https://github.com/allthesignals/future-mobility-responses/commit/c8adcdc1f1d047850c27d11a56c14ef112cb6cf5:

```javascript
// devDependencies
"@ember-decorators/babel-transforms": "^2.0.0",
"ember-cli": "~3.5.0",
"ember-decorators": "3.0.2",
 "ember-source": "~3.5.0",
```

After `ember install ember-concurrency-decorators`, my project console warns me:
> WARNING: The 'ember-concurrency-decorators' addon is using an older version of @ember-decorators/babel-transforms that uses stage 1 decorators. Please update to @ember-decorators/babel-transforms@v3.1 to remove your dependency on legacy decorators.

My browser console tells me:
```
Uncaught TypeError: Cannot read property 'writable' of undefined
```

What is happening? How is this still using stage 1 decorators?

**Comment by lolmaus** [comment]
This is likely caused by `ember-concurrency-decorators` pulling a newer major version of `ember-decorators` 3, which in turn pulls a newer major version of `ember-cli-babel` 7.

And your Ember CLI setup is likely on `ember-cli-babel` 6.

**Comment by buschtoens** [comment]
The explanation given by @lolmaus makes a lot of sense to me. I'll upgrade this addon to the most recent version of `ember-decorators` and try to keep the version range as wide as possible.

**Comment by buschtoens** [comment]
I published `1.0.0-beta.1` which drops stage 2 support and reverts back to legacy / stage 1 decorators in accordance with the decorators support RFC.
